### PR TITLE
[NETBEANS-5074] - cleanup ArrayList warnings..

### PR DESCRIPTION
**Pull request by BradWalker**
This work cleans up a bunch of warnings surrounding the use of ArrayList..

For example:

   [repeat] /home/bwalker/src/netbeans/platform/core.multitabs/src/org/netbeans/core/multitabs/impl/TabLayoutManager.java:245: warning: [unchecked] unchecked conversion
   [repeat]             List<Integer>[] rowIndexes = new ArrayList[rowCount];
   [repeat]                                          ^
   [repeat]   required: List<Integer>[]
   [repeat]   found:    ArrayList[]

ActionsManager.java was especially involved since one change would require another. And keeping track of
that, so as not to break things, was hard.

### CODE REVIEW

File: java/java.sourceui/src/org/netbeans/modules/java/source/ui/OpenProjectFastIndex.java [hunk 1]
```diff
@@ -550,7 +550,7 @@ private PI getCacheInfo(Project p) {
         private final int         size;
         private final Project     project;
         
-        NameIndex(Project p, FileObject root, String files, ArrayList indices) {
+        NameIndex(Project p, FileObject root, String files, List indices) {
```
Comment (mklaehn) [inline java/java.sourceui/src/org/netbeans/modules/java/source/ui/OpenProjectFastIndex.java:553]:
```suggestion
        NameIndex(Project p, FileObject root, String files, List<Object[]> indices) {
```

**Review by mklaehn — APPROVED** [review]
Nice work.
Commented on a place where the type of the list could be added.

File: java/java.sourceui/src/org/netbeans/modules/java/source/ui/OpenProjectFastIndex.java (hunk 1 above)
Comment (BradWalker) [reply java/java.sourceui/src/org/netbeans/modules/java/source/ui/OpenProjectFastIndex.java]: Hey @mklaehn , thanks for the sharp eyes!

It's been changed and integrated..

**Review by mklaehn — APPROVED** [review]
thank you for the work

**Pull request merged**